apis/Google.Cloud.Logging.Console/Google.Cloud.Logging.Console.Tests/GoogleCloudConsoleFormatterTest.cs

```diff
@@ -187,6 +187,128 @@ public void Log_TraceInformationIsLogged()
         Assert.Equal(expectedJson, actualJson);
     }
 
+    public class SimpleAugmenter : IGoogleCloudConsoleLogAugmenter
+    {
+        /// <summary>
+        /// Augment with a simple key-value pair.
+        /// </summary>
+        public void AugmentFormattedLogEntry<TState>(in LogEntry<TState> logEntry, IExternalScopeProvider scopeProvider, Utf8JsonWriter writer)
+        {
+            writer.WriteString("simple_key", "simple_value");
+        }
+    }
+    private readonly GoogleCloudConsoleFormatterOptions simpleOptions = new GoogleCloudConsoleFormatterOptions { LogAugmenter = new SimpleAugmenter() };
```

Comment:
I would move all of these into their respective tests, I don't think you are sharing any.